app/frontend/src/components/MarkdownViewer/MarkdownViewer.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { marked } from 'marked';
+import styles from "./MarkdownViewer.module.css";
+import { Spinner, SpinnerSize, MessageBar, MessageBarType, Link, IconButton } from "@fluentui/react";
+
+interface MarkdownViewerProps {
+    src: string;
+}
+
+export const MarkdownViewer: React.FC<MarkdownViewerProps> = ({ src }) => {
+    const [content, setContent] = useState<string>("");
+    const [isLoading, setIsLoading] = useState<boolean>(true);
+    const [error, setError] = useState<Error | null>(null);
+
+    const removeAnchorLinks = (html: string) => {
```

Comment:
Can you comment as to why this is needed?

Reply:
Added